client/src/main/java/io/spine/client/TopicFactory.java

```diff
@@ -83,7 +83,7 @@ public Topic allOf(Class<? extends Message> targetType) {
     }
 
     /**
-     * Creates a {@link Topic} for the specified {@link Target}, updates for which will include
+     * Creates a {@link Topic} for the specified {@link Target}; updates for it will include
```

Comment:
Please split into 2 sentences. I'm not sure if the `javadoc` tool will or will not cut the doc "summary" at a `;`.